test/functional/teardown.js

```diff
@@ -0,0 +1,5 @@
+// const chalk = require('chalk');
+module.exports = async () => {
+  // console.log(chalk.green('Teardown Verdaccio Functional'));
```

Comment:
why commented out code?

Reply:
Because I'm really lazy, I will correct it